app/assets/stylesheets/govuk_publishing_components/components/_attachment.scss

```diff
@@ -0,0 +1,43 @@
+$thumbnail-width: 99px;
+$thumbnail-height: 140px;
+$thumbnail-border: 5px;
+$thumbnail-border-colour: rgba(11, 12, 12, .1);
```

Comment:
Very minor comment - this is using an RGBA value where it isn't really necessary - since I think this will always be displayed on a white background? It could be changed to `rgb(230, 230, 230);` and appear the same, but older browsers (IE8) would be able to display it.

Reply:
I changed this to the rgb value and for some reason the shadow disappeared 😱

Reply:
Spoke with @alex-ju about this and he suggested we stick with using RGBA here; it turns out we'll be rendering this component in Content Publisher on a light grey background.